### DISCUSSION TITLE
Can Fluid's Shareable Widgets/UI be re-used across Non-MS-based apps ?

### DISCUSSION THREAD

**Discussion by varuncomp** [microsoft/FluidFramework#9353] [General]
As per the documentation, I understand that Fluid framework is a set of libraries which help the developer to build applications that support real-time data sync. MS-based ecosystem of products(especially Office 365 apps) use the fluid framework to build "Fluid components" that are live components which can be edited in-line and support real-time data sync. However, I'm curious to know if Fluid framework also encompasses re-usable shareable UI Widgets/Fluid components(The ones used in MS Teams is attached as a screenshot for reference) that can be used across different web-applications, which further reduces the efforts on the developer. 

Can I expect these Fluid components to be available as open-source components ?
![Screenshot 2022-03-04 at 2 50 02 PM](https://user-images.githubusercontent.com/40627855/156735593-3b861fbc-6451-436b-b6d5-7e1d18b98303.png)

**Comment by tylerbutler** [comment]
One important thing to note is that as a data library, Fluid does not have strong affinity to a particular UI/UX technology or library. The Framework itself is very data-focused. Having said that, the O365/Teams/Loop experiences that Microsoft has built on top of Fluid's underlying infrastructure _is_ UX focused and uses the framework to promote portability, basically by creating contracts between layers of the system, some of which control how a component renders. This enables different hosting "surfaces" to load Fluid data objects and containers, render them, and interact with them.

All of that is built on top of the core framework, but isn't part of it. It's unclear what the extensibility of those high level layers will be long-term.

**Reply by varuncomp** [reply]
Thanks for confirming my understanding and providing your insights Tyler!
